max_iteration: 4
img_enable: 1
lidar_enable: 1

point_filter_num: 4
filter_size_surf: 0.3
filter_size_map: 0.3

common:
  img_topic: "/camera/image_raw"
  lid_topic: "/velodyne_points"
  imu_topic: "/imu/data"

preprocess:
    lidar_type: 2 # Livox Avia LiDAR
    scan_line: 16
    blind: 0.5 # blind x m disable
    timestamp_unit: 3

mapping:
    gyr_cov: 0.05
    acc_cov: 0.1
    b_gyr_cov: 0.0001
    b_acc_cov: 0.0001
    laser_point_cov: 0.0001
    det_range: 450.0
    extrinsic_est_en: false
    extrinsic_T: [ 0.04165, 0.02326, -0.0284 ]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]
    Rcl: [0.00162756,-0.999991,0.00390957,
         -0.0126748,-0.00392989,-0.999912,
          0.999918,0.00157786,-0.012681]
    Pcl: [0.0409257, 0.0318424, -0.0927219]

camera:
    debug: 0
    visual_point_cov: 100 # 00: 100, 01: 200
    cam_width: 640
    cam_height: 512
    cam_fx: 431.712
    cam_fy: 431.7085
    cam_cx: 320.3404
    cam_cy: 259.1696
    grid_size: 60
    patch_size: 8

pcd_save:
    pcd_save_en: false
